app/controllers/kudos_controller.rb

```diff
@@ -43,24 +43,32 @@ def index
   # Creates a new Kudo based on current user and specified receiver & body
   #
   # Required params:
-  #   :receiver_email [email of receiver]
-  #   :body           [body of kudo]
+  #   :receiver_emails [array of emails of receivers]
+  #   :body            [body of kudo]
   def create
     giver_id = current_user.id
-    receiver_email = params[:kudo][:receiver_email].split(',').first.strip
+    receiver_emails = params[:kudo][:receiver_emails]
+    receivers = User.where(email: receiver_emails)
 
-    unless (receiver = User.find_by(email: receiver_email))
-      if !(/@factual.com$/ =~ receiver_email)
-        return render(json: { errors: "recipient with email #{receiver_email.inspect} is not part of this organization"}, status: :not_found)
+    uncreated_emails = receiver_emails - receivers.pluck(:email)
+    uncreated_receivers = uncreated_emails.map do |email|
+      if !(/@factual.com$/ =~ email)
+        return render(
+          json: {
+            errors: "recipient with email #{email.inspect} is not part of this organization"
+          },
+          status: :not_found
+        )
       end
-      #return render json: { errors: "recipient with email #{receiver_email.inspect} could not be found" }, status: :not_found
-      receiver = User.new(name: receiver_email, email: receiver_email)
-      receiver.save
+      { name: email, email: email }
     end
 
+    new_receivers = User.create!(uncreated_receivers)
+    receiver_ids = receivers.ids.concat new_receivers.map(&:id)
+
     kudo = Kudo.new(
       giver_id: giver_id,
-      receiver_ids: [receiver.id],
+      receiver_ids: receiver_ids,
```

Comment:
Same confusion as other PR (`receiver_ids` vs `receivers`)!

Reply:
Another maybe less confusing way to do this is:
```ruby
kudo = Kudo.new
kudo.giver = current_user
kudo.receivers = receivers.concat new_receiversa
kudo.body = kudo_params[:body]
kudo.save
```
What do you think?

Reply:
Actually, I think I'm going to do `kudo.receivers = receivers`, since that will let nate do a `receivers.each { |r| r.notfiy(kudo) }` more easily.